src/components/sections/landing/Landing_Hero.js

```diff
@@ -64,6 +64,9 @@ export default function LandingHero() {
           _hover={{
             color: "#AE58BD",
           }}
+          onClick={() => {
+            window.open("https://perped.gitbook.io/");
```

Comment:
@br0wnD3v the main app shouldn't open in a new window.

Reply:
You can look more into chakra's `<Link>` - https://chakra-ui.com/docs/components/link

See if we can use this with a router

Reply:
This should also fix the button animation issue.

```
<Link href="https://app.perped.com/"><CustomButton text="Enter App" /></Link>
```

Reply:
Also, use the same Link component for social links. Pick them up from discord

Reply:
Everything is redirected using `<Link>` already. `<Button>` was used because of a styling issues and nothing more :) . Fixed the Main app redirect to open up on the same page and any other Social/Gitbook link to be opened up in a new tab.